documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -240,54 +240,54 @@
+      <input name="thin_walled" type="boolean" value="false" uiname="Thin walled"/>
+      <input name="normal" type="vector3" uiname="Normal Input">
         <geomprop name="normal" type="vector3" space="world"/>
       </input>
-      <input name="tangent" type="vector3">
+      <input name="tangent" type="vector3" uiname="Tangent Input">
         <geomprop name="tangent" type="vector3" space="world"/>
       </input>
-      <input name="coat" type="float" value="0" />
-      <input name="coat_color" type="color3" value="1, 1, 1" />
-      <input name="coat_roughness" type="float" value="0.1" />
-      <input name="coat_IOR" type="float" value="1.5" />
-      <input name="coat_normal" type="vector3">
+      <input name="coat" type="float" value="0" uiname="Coat" uifolder="Coat"/>
+      <input name="coat_color" type="color3" value="1, 1, 1" uiname="Coat Color" uifolder="Coat"/>
+      <input name="coat_roughness" type="float" value="0.1" uiname="Coat Roughness" uifolder="Coat"/>
+      <input name="coat_IOR" type="float" value="1.5" uiname="Coat Index of Refraction" uifolder="Coat"/>
+      <input name="coat_normal" type="vector3" uiname="Coat normal" uifolder="Coat">
         <geomprop name="normal" type="vector3" space="world"/>
       </input>
-      <input name="sheen" type="float" value="0" />
-      <input name="sheen_color" type="color3" value="1, 1, 1" />
-      <input name="sheen_roughness" type="float" value="0.3" />
-      <input name="thin_film_thickness" type="float" value="0" />
-      <input name="thin_film_IOR" type="float" value="1.5" />
-      <input name="emission" type="float" value="0" />
-      <input name="emission_color" type="color3" value="1, 1, 1" />
-      <input name="opacity" type="color3" value="1, 1, 1" />
-      <input name="caustics" type="boolean" value="false" />
-      <input name="internal_reflections" type="boolean" value="true" />
-      <input name="exit_to_background" type="boolean" value="false" />
-      <input name="indirect_diffuse" type="float" value="1" />
-      <input name="indirect_specular" type="float" value="1" />
+      <input name="sheen" type="float" value="0" uiname="Sheen" uifolder="Sheen"/>
+      <input name="sheen_color" type="color3" value="1, 1, 1" uiname="Sheen Color" uifolder="Sheen"/>
+      <input name="sheen_roughness" type="float" value="0.3" uiname="Sheen Roughness" uifolder="Sheen"/>
+      <input name="thin_film_thickness" type="float" value="0" uiname="Thin Film Thickness" uifolder="Thin Film"/>
+      <input name="thin_film_IOR" type="float" value="1.5" uiname="Thin Film Index of Refractin" uifolder="Thin Film"/>
+      <input name="emission" type="float" value="0" uiname="Emission" uifolder="Emission"/>
+      <input name="emission_color" type="color3" value="1, 1, 1" uiname="Emission Color" uifolder="Emission"/>
+      <input name="opacity" type="color3" value="1, 1, 1" uiname="Opacity" />
+      <input name="caustics" type="boolean" value="false" uiname="Caustics" />
```

Comment:
caustics, internal_reflections, exit_to_background and the indirect_diffuse/specular can be placed into an "Advanced" uifolder.